### ISSUE TITLE
Cubic centimeters parses to deciliter

### ISSUE THREAD

**Issue by airarrazaval** [nielstron/quantulum3#191]
**Describe the bug**
Cubic centimeter using `cm3` or `cm^3` is parsed to **deciliter**.

**To Reproduce**
```python
from quantulum3.parser import parse

values = ['1 cc', '2 ml', '3 cm3', '4 cm^3']

for value in values:
  print(parse(value)[0].to_spoken())
```

Output:

```
one cubic centimetre
two cubic centimetres
three decilitres
four decilitres
```

**Expected behavior**
`cm3` or `cm^3` should be parsed to `cc`

**Additional information:**
 - Python Version: `3.8.10`
 - Classifier activated/ sklearn installed: `no`
 - OS: `Ubuntu`
 - Version: `20.04 LTS`

**Comment by nielstron** [comment]
Thanks for posting this issue! I found the root of this issue and a number of related problems in the unit database and removed them in https://github.com/nielstron/quantulum3/commit/48818bdab47aa887d45b3040128ebe7a10b0e66e . As soon as this passes, I will publish it in a new version of quantulum3

**Comment by nielstron** [comment]
fixed in release 0.7.9
